Settings.js

```diff
@@ -306,6 +306,17 @@ function avtt_settings() {
 			],
 			defaultValue: false
 		}
+		,
+		{
+			name: "highlightScrolling",
```

Comment:
I think we should note it only affects the combat tracker and token pings. 

Find token buttons for example should always center (you haven't changed those I don't think which is good). 

Maybe we consider having default value false based on feedback in beta. Either way is good with me though. We can consider moving the setting to a toggle in UI after too.